assets/archers_paradox/lang/zh_tw.json

```
@@ -0,0 +1,49 @@
+{
+  "effect.archers_paradox.challenge_complete": "挑戰完成",
+  "effect.archers_paradox.challenge_miss": "挑戰失敗",
+  "effect.archers_paradox.challenge_streak": "弓箭手挑戰",
+
+  "effect.archers_paradox.training_miss": "中斷連擊",
+  "effect.archers_paradox.training_streak": "連擊",
+
+  "itemGroup.archers_paradox": "弓箭手悖論",
+
+  "item.archers_paradox.quiver": "箭袋",
+
+  "item.archers_paradox.blaze_arrow": "烈焰箭矢",
+  "item.archers_paradox.blaze_arrow.desc": "箭矢撞擊時將附近方塊與生物點燃。",
+  "item.archers_paradox.challenge_arrow": "挑戰者箭矢",
+  "item.archers_paradox.challenge_arrow.desc": "不造成傷害，用於測試弓箭手的技巧。讓每一發都更有難度。",
+  "item.archers_paradox.diamond_arrow": "鑽石箭矢",
+  "item.archers_paradox.diamond_arrow.desc": "增加傷害，射穿一個目標",
+  "item.archers_paradox.displacement_arrow": "置換箭矢",
+  "item.archers_paradox.displacement_arrow.desc": "創造一個暫時的區域，將事物傳送走。",
+  "item.archers_paradox.ender_arrow": "終界箭矢",
+  "item.archers_paradox.ender_arrow.desc": "傳送被擊中者到身旁，或傳送到箭矢所落之處。",
+  "item.archers_paradox.explosive_arrow": "爆炸箭矢",
+  "item.archers_paradox.explosive_arrow.desc": "箭矢撞擊時產生爆炸。",
+  "item.archers_paradox.frost_arrow": "冰霜箭矢",
+  "item.archers_paradox.frost_arrow.desc": "箭矢擊中時凍結附近方塊和凍僵目標。",
+  "item.archers_paradox.glowstone_arrow": "螢光石箭矢",
+  "item.archers_paradox.glowstone_arrow.desc": "讓周圍稍微變亮。",
+  "item.archers_paradox.lightning_arrow": "閃電箭矢",
+  "item.archers_paradox.lightning_arrow.desc": "箭矢撞擊時呼喚下雷。",
+  "item.archers_paradox.phantasmal_arrow": "幻影箭矢",
+  "item.archers_paradox.phantasmal_arrow.desc": "直線飛行，刺穿一切。",
+  "item.archers_paradox.prismarine_arrow": "海磷箭矢",
+  "item.archers_paradox.prismarine_arrow.desc": "改進箭矢在水下飛行。",
+  "item.archers_paradox.quartz_arrow": "石英箭矢",
+  "item.archers_paradox.quartz_arrow.desc": "稍微增加傷害與擊退。",
+  "item.archers_paradox.redstone_arrow": "紅石箭矢",
+  "item.archers_paradox.redstone_arrow.desc": "對半徑範圍內的東西進行短時間的充能。",
+  "item.archers_paradox.shulker_arrow": "界伏箭矢",
+  "item.archers_paradox.shulker_arrow.desc": "讓目標回家。",
+  "item.archers_paradox.slime_arrow": "史萊姆箭矢",
+  "item.archers_paradox.slime_arrow.desc": "箭矢可彈跳好幾次。低傷害高擊退。",
+  "item.archers_paradox.spore_arrow": "孢子箭矢",
+  "item.archers_paradox.spore_arrow.desc": "擴散菌類的光榮。",
```

```suggestion
  "item.archers_paradox.spore_arrow.desc": "傳播菌類的榮耀。",
```
既然是孢子，感覺傳播比較好。
至於光榮，感覺一般都是當形容詞用，不宜放在「的」後面